### PR TITLE
November 2021 newsletter

### PR DESCRIPTION
**Pull request by MILeach**
The Sheffield RSE November 2021 newsletter

### CODE REVIEW

File: _newsletters/2021-11-15-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+## University of Sheffield Research Software Engineering Team Newsletter November 2021
+
+Hi everyone!
+
+Welcome to the monthly newsletter from the [Research Software Engineering Team](https://rse.shef.ac.uk/) at The University of Sheffield.
+
+## Opportunities
+- [Research Software Engineer](https://www.jobs.ac.uk/job/CKR689/research-software-engineer). Come and join us! This is an open-ended position at the University of Sheffield Research Software Engineering team. The role is ideal for someone with a passion for research and for good software engineering looking to use their skills in collaborative software development and reproducible research to make an impact on research projects across the whole of the University of Sheffield and beyond. Closing date 6th December 2021.
```
Comment (Robadob) [inline _newsletters/2021-11-15-newsletter.md]: > with a passion for research and for good software engineering

Reword

File: _newsletters/2021-11-15-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    - **As an author:** [Submit your paper for reproduction and review](https://www.reprohack.org/paper/)!
+    - **As a participant:** [Join on us on this celebratory day](https://bit.ly/reprohack-hub-launch) and get practical experience in reproducibility with real published materials!
+- **GitHub Citation Files:** GitHub have introduced support for [citation files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files). These are simple to use and ensure users of your work can easily cite it correctly!
+
+
+### Selected Articles
+
+This is a small selection of articles from the Software Sustainability Institute which may be of interest!
```
Comment (Robadob) [inline _newsletters/2021-11-15-newsletter.md]: This is a small selection of recent* articles?

File: _newsletters/2021-11-15-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+- [Don’t be afraid - anyone can learn to code](https://www.software.ac.uk/blog/2021-11-02-dont-be-afraid-anyone-can-learn-code?mc_cid=94afd68d24&mc_eid=f09318bc47)
+- [Using Python to double check your work](https://www.software.ac.uk/blog/2021-11-01-using-python-double-check-your-work?mc_cid=94afd68d24&mc_eid=f09318bc47)
+- [5 Popular Myths of Learning to Code](https://www.software.ac.uk/blog/2021-11-01-guide-5-popular-myths-learning-code?mc_cid=94afd68d24&mc_eid=f09318bc47)
+
+### Training 
+
+- There's a bunch of **upcoming free training courses from the Hartree Centre** including:
+  - Mixed language programming – From Python to C | 9 Dec 2021 : [https://www.cvent.com/c/express/1e0034b8-643f-4862-a8a3-3db7575cd88d](https://www.cvent.com/c/express/1e0034b8-643f-4862-a8a3-3db7575cd88d)
```
Comment (Robadob) [inline _newsletters/2021-11-15-newsletter.md]: The link goes to 'this event is closed', so not worth advertising as it's presumably sold out.

File: _newsletters/2021-11-15-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+  For more details on the courses, check out their [Accessible supercomputing​, data analytics and AI training with industry context](https://www.hartree.stfc.ac.uk/Pages/Explain.aspx?utm_source=Twitter&utm_medium=social&utm_campaign=Orlo) page.
```
Comment (Robadob) [inline _newsletters/2021-11-15-newsletter.md]: Nothing currently bookable, either full or TBA.

**Review by Robadob — CHANGES_REQUESTED** [review]
Take a look at my inline comments, otherwise reads well.

File: _newsletters/2021-11-15-newsletter.md (hunk 1 above)
Comment (bobturneruk) [reply _newsletters/2021-11-15-newsletter.md]: Why?

Comment (bobturneruk) [reply _newsletters/2021-11-15-newsletter.md]: Please use this link https://rse.shef.ac.uk/blog/2021-11-12-rse-roles/

**Pull request merged**